tests/plugins/test_flatpak_create_oci.py

```diff
@@ -364,9 +364,10 @@ class MockSource(object):
 
 
 class X(object):
-    image_id = "xxx"
-    source = MockSource()
-    base_image = ImageName(repo="qwe", tag="asd")
+    def __init__(self):
```

Comment:
If you are changing this, maybe rename X to MockBuilder - I originally copied this from other tests, but I think the use of X for a classname is pretty confusing and unreadable.

Reply:
Fixed